### PR TITLE
changes for icarus: add an option to extract pulse start time

### PR DESCRIPTION
**Pull request by drinkingkazu**

### CODE REVIEW

**Comment by lgarren** [comment]
trigger build

**Comment by knoepfel** [comment]
Will approve once we have L2 approval.

**Comment by SFBayLaser** [comment]
+1

File: larana/OpticalDetector/OpHitFinder/OpHitAlg.cxx [hunk 1]
```diff
@@ -66,12 +67,13 @@ namespace opdet {
...
                detinfo::DetectorClocksData const& clocksData,
-               calib::IPhotonCalibrator const& calibrator)
+               calib::IPhotonCalibrator const& calibrator,
+	       bool use_start_time)
   {
 
     if (pulse.peak < hitThreshold) return;
 
-    double absTime = timeStamp + pulse.t_max * clocksData.OpticalClock().TickPeriod();
+    double absTime = timeStamp + clocksData.OpticalClock().TickPeriod() * (use_start_time ? pulse.t_start : pulse.t_end);
```
Comment (knoepfel) [inline larana/OpticalDetector/OpHitFinder/OpHitAlg.cxx]: Shouldn't the backwards-compatible change be:
```diff
double absTime = timeStamp + clocksData.OpticalClock().TickPeriod() *
- (use_start_time ? pulse.t_start : pulse.t_end);
+ (use_start_time ? pulse.t_start : pulse.t_max);
```

Comment (drinkingkazu) [reply larana/OpticalDetector/OpHitFinder/OpHitAlg.cxx]: ah yes! Sorry for this mistake in rush.

**Comment by lgarren** [comment]
trigger build

File: larana/OpticalDetector/OpHitFinder/OpHitAlg.cxx (hunk 1 above)
Comment (knoepfel) [reply larana/OpticalDetector/OpHitFinder/OpHitAlg.cxx]: No problem, Kazu.

**Comment by lgarren** [comment]
It looks like the CI was OK.  There were some hiccups due to work on the database server today.

**Comment by lgarren** [comment]
approve

**Pull request merged**